packages/salesforcedx-vscode-apex/test/vscode-integration/commands/apexTestSuite.test.ts

```diff
@@ -10,40 +10,37 @@ import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode';
 import { expect } from 'chai';
 import { createSandbox } from 'sinon';
 import * as vscode from 'vscode';
+import {
+  ApexTestQuickPickItem,
+  TestType
+} from '../../../src/commands/apexTestRun';
 import {
   ApexTestSuiteOptions,
   TestSuiteBuilder,
   TestSuiteCreator,
   TestSuiteSelector
 } from '../../../src/commands/apexTestSuite';
-import {
-  ApexTestQuickPickItem,
-  TestType
-} from '../../../src/commands/forceApexTestRun';
 import { workspaceContext } from '../../../src/context';
 
 const sb = createSandbox();
 
-describe('Test Suite Selector', async () => {
+describe('Test Suite Selector', () => {
```

Comment:
Why deleting the async action? I am thinking this might potentially trigger the failure of e2e tests in windows.